test/types-multipart.spec.js

```diff
@@ -229,6 +229,25 @@ describe('types-multipart', () => {
       ],
       what: 'Empty content-type and empty content-disposition'
     },
+    {
+      config: {
+        isPartAFile: (fieldName) => (fieldName !== 'upload_file_0')
+      },
+      source: [
```

Comment:
can you add another file with a different name to ensure that one of them still gets picked as a file?

Reply:
or it's generally not possible to mix files and fields?

Reply:
it is possible. modified unit test.